### PR TITLE
Add Thai Gen AI (Dolly) dataset

### PR DESCRIPTION
**Pull request by manandey**
All tests are passing. c.c @shayne-longpre

### CODE REVIEW

**Review by wwbrannon — APPROVED** [review]
LGTM!

File: data_summaries/Thai Gen AI (Dolly).json [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+        "Task Categories": [
+            "Question Answering"
+        ],
+        "Format": [
+            "Zero-shot"
+        ],
+        "Text Sources": [
+            "crowdsourced"
+        ],
+        "Model Generated": [],
```
Comment (shayne-longpre) [inline data_summaries/Thai Gen AI (Dolly).json:26]: Google Cloud Translation API on each of these

File: data_summaries/Thai Gen AI (Dolly).json [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+        "Creators": [],
+        "Licenses": [
+            {
+                "License": "CC BY-SA 3.0",
+                "License URL": "https://huggingface.co/datasets/Thaweewat/databricks-dolly-15k-th#summary"
+            }
```
Comment (shayne-longpre) [inline data_summaries/Thai Gen AI (Dolly).json:32]: Google Cloud and original dolly license?

File: data_summaries/Thai Gen AI (Dolly).json [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+        ],
+        "License Notes": "",
+        "License Verified By": "Manan",
+        "Dataset Filter IDs": [
+            "open_qa",
+            "general_qa"
+        ],
+        "Derived from Datasets": [],
```
Comment (shayne-longpre) [inline data_summaries/Thai Gen AI (Dolly).json:40]: derived from original Dolly

**Review by shayne-longpre — COMMENTED** [review]
Some minor additions

**Pull request merged**